person.rb

```diff
@@ -0,0 +1,33 @@
+require './nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age, :id
+  attr_reader :rentals
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+    super()
+  end
+
+  def can_use_services?
+    @parent_permission || of_age?(@age)
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] good job creating this method, however, it should look a little different. please review [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more details.